courses/free-courses-kk.md

```diff
@@ -4,6 +4,7 @@
 * [HTML/CSS](#html/css)
 * [Javascript](#javascript)
 * [PHP](#php)
+* [Python](#python)
```

Comment:
:information_source: It's usually not helpful to add a category for a single resource. :twisted_rightwards_arrows: Try to...
- ... find another category matching their topics.
- ... see if there are other books in our lists or third-party that would fit into your category (this fact contributes to consolitate it).

**:rocket: But is a fact that there aren`t more options**